CHANGELOG.md

```diff
@@ -24,8 +24,10 @@
 #### Bug Fixes
 
 - Fixed a bug in local testing mode that caused a column to contain None when it should contain 0
+- Fixed a bug in StructField.from_json that prevented TimestampTypes with tzinfo from being parsed correctly.
 - Fixed a bug in `StructField.from_json` that prevented TimestampTypes with tzinfo from being parsed correctly.
```

Comment:
this needs a fix?

Reply:
ohh, this is issue from rebase, will change